Application/Transfer/Import/MLS/Service.php

```diff
@@ -0,0 +1,945 @@
+                                        );
+
+                                        Address::useService()->insertAddressToPerson(
+                                            $tblPersonMother, $streetName, $streetNumber, $studentCityCode, $cityName, '', ''
+                                        );
+
+                                        $countMother++;
+                                    } else {
+
+                                        Relationship::useService()->insertRelationshipToPerson(
+                                            $tblPersonMotherExists,
+                                            $tblPerson,
+                                            $tblRelationshipTypeCustody,
+                                            '',
+                                            $genderSetting == 'Weiblich' ? 1 : 2
+                                        );
+
+                                        $error[] = 'Zeile: ' . ($RunY + 1) . ' Die Mutter wurde nicht angelegt,
+                                            da schon eine Person mit gleichen Namen und gleicher PLZ existiert.
+                                            Der Schüler wurde mit der bereits existierenden Person verknüpft';
+
+                                        $countMotherExists++;
+                                    }
+                                }
+                            }
+                        }
+                    }
+
+                    return
+                        new Success('Es wurden ' . $countStudent . ' Schüler erfolgreich angelegt.') .
+                        new Success('Es wurden ' . $countFather . ' Väter erfolgreich angelegt.') .
+                        ($countFatherExists > 0 ?
+                            new Warning($countFatherExists . ' Väter exisistieren bereits.') : '') .
+                        new Success('Es wurden ' . $countMother . ' Mütter erfolgreich angelegt.') .
+                        ($countMotherExists > 0 ?
+                            new Warning($countMotherExists . ' Mütter exisistieren bereits.') : '')
+                        . new Layout(new LayoutGroup(new LayoutRow(new LayoutColumn(
+                            new Panel(
+                                'Fehler',
+                                $error,
+                                Panel::PANEL_TYPE_DANGER
+                            )
+                        ))));
+
+                } else {
+                    return new Warning(json_encode($Location)) . new Danger(
+                            "File konnte nicht importiert werden, da nicht alle erforderlichen Spalten gefunden wurden");
+                }
+            }
+        }
+
+        return new Danger('File nicht gefunden');
+    }
+
+    /**
+     * @param IFormInterface|null $Form
+     * @param UploadedFile $File
+     *
+     * @return IFormInterface|Danger|string
+     */
+    public function createStaffsFromFile(IFormInterface $Form = null, UploadedFile $File = null)
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Incorrect spacing between argument "$Form" and equals sign; expected 0 but found 1](https://app.codacy.com/manual/gerdchristian-kunze/SPHERE-Framework/pullRequest?prid=5646354)